### DISCUSSION TITLE
How did you learn react

### DISCUSSION THREAD

**Discussion by blgoreuxe** [blueedgetechno/win11React#272] [General]
i have been wanting to learn react, currently i have been taking a scrimba course.

how did u learn react, blue?

**Comment by blueedgetechno** [comment]
I wanted to learn React to make a plugin in Atom text editor of my choice. So I watched a 30 min tutorial for starters, and created the plugin.

Then I created more jsvascript games with React, got an internship as a Front End developer, learned Advanced concepts like state management, Redux, Action handling, Error Handling etc.

After that I finally implemented the knowledge in this project.
